packages/node-patches/readme.md

```diff
@@ -0,0 +1,52 @@
+# @bazel/node-patches
+
+Runtime modifications to nodejs internals to help node/npm distributed programs run in bazel
+
+## fs
+
+patch any `fs` like object such that symlinks that point outside a specified directory seem to be their targets instead of links
+
+### example
+
+```js
+const fs = require('fs')
+const path = require('path')
+const patcher = require('@bazel/node-patches')
```

Comment:
It would be convient to hook this up directly into `internal/node/node_loader.js` script in a similar way that `install_source_map_support` is glued in

https://github.com/bazelbuild/rules_nodejs/blob/71635716abb1ec6d4466c5d007e7a766e77ab802/internal/node/node_loader.js#L61

https://github.com/bazelbuild/rules_nodejs/blob/bc848f9d58175dad727b3285f36e7db88e5aeb48/internal/node/node.bzl#L112

Have you thought about how to make using this script a bit more ergonomic for consumers?

Reply:
i intend to inject it directly in https://github.com/bazelbuild/rules_nodejs/blob/master/internal/node/bazel_require_script.js
in a follow up pr. This is a more production ready rewrite of this hack version that just eliminated nodejs's ability to detect symlinks.

I'm going to meet with @gregmagolan today to make sure how it lands makes sense from a bazel perspective.